frontend/server/src/DAO/SchoolOfTheMonth.php

```diff
@@ -101,6 +101,40 @@ public static function calculateSchoolsOfMonthByGivenDate(
         return self::calculateSchoolsOfMonth($startTime, $endTime, $rowcount);
     }
 
+    /**
+     * Gets all the best schools based on the month
+     * of a certain date.
+     *
+     * @return list<array{school_id: int, rank: int, name: string, country_id: string}>
+     */
+    public static function getSchoolsOfCertainMonth(
```

Comment:
```suggestion
    public static function getMonthlyList(
```

por consistencia con `\OmegaUp\DAO\CoderOfTheMonth::getMonthlyList()`